lightning/src/blinded_path/payment.rs

```diff
@@ -150,6 +153,75 @@ pub(super) fn blinded_hops<T: secp256k1::Signing + secp256k1::Verification>(
 	utils::construct_blinded_hops(secp_ctx, pks, tlvs, session_priv)
 }
 
+pub(super) fn compute_payinfo(
+	intermediate_nodes: &[(PublicKey, ForwardTlvs)], payee_tlvs: &ReceiveTlvs, htlc_maximum_msat: u64
+) -> Result<BlindedPayInfo, ()> {
+	let mut curr_base_fee: u128 = 0;
```

Comment:
I kept the `htlc_min` parts as `u128` since `BlindedPayInfo::htlc_minimum_msat` is u64, so it leaves room for computing the proportional fee without an overflow. But it's really unlikely we'd encounter a min htlc high enough to overflow there, so lmk if you'd prefer to switch those over too.